### DISCUSSION TITLE
windows的标准版安装后闪退，beta版不能授权

### DISCUSSION THREAD

**Discussion by feuyeux** [getlantern/lantern-forum-cn#738] [General]
尽快修正windows标准版，解决闪退问题。

**Comment by ycjdixon** [comment]
@feuyeux
这个我们已经反映好几个月了。他们一个星期开一次会，然后就没有然后了。技术团队平时不上班的。你可以去找一下免费的VPN。

**Reply by MaxMKMax** [reply]
@feuyeux 您好. beta测试版原本就无法用授权码. 您要不是看官方版本吧. 谢谢.

**Reply by MaxMKMax** [reply]
@feuyeux 
还依然闪退吗? 请问您使用哪一个设备? 和哪一个版本? 也帮我提交各应用内报告吧然后给我Log. 谢谢!

**Reply by feuyeux** [reply]
Win11最新版本 蓝灯也是最新下载版本 
外星人笔记本
![IMG_20240731_000231.jpg](https://github.com/user-attachments/assets/83d6b5ac-d72e-431d-b46a-350c63d5cc5a)



日志路径在哪

**Reply by Wolffy233** [reply]
> @feuyeux 您好. beta测试版原本就无法用授权码. 您要不是看官方版本吧. 谢谢.

不能用授权码，不能关闭入口么？这不是明摆着误导用户么，标准版windows上打开直接被杀死，没法用怎么解决？我是真想骂你们这些人，付费的产品就好好做，你们这个价格也不便宜，去隔壁看看expressvpn比你们便宜，我刚下单了两年的蓝灯，你们这标准版windows上闪退，预览版不能授权，邮件登录也半天没反应我看就续费还挺快的，真是越做越垃圾

**Reply by MaxMKMax** [reply]
@boatImpish 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by feuyeux** [reply]
> @feuyeux
> 这个我们已经反映好几个月了。他们一个星期开一次会，然后就没有然后了。技术团队平时不上班的。你可以去找一下免费的VPN。

可是我已经付费给你们了 你这样的回复作为客户 我无法接受 你们一定要解决问题 在修复期间 应当给客户暂停付费 延期过期

这是影响你们品牌形象的事情

**Reply by MaxMKMax** [reply]
@feuyeux 非常抱歉, 完全明白您的想法. 您再说明一下您遇到的问题, 我再次尽快地让他们解决问题. 真的非常不好意思影响了您的体验感. 您的反馈非常的好, 我们会尽所能解决用户的问题. 再次和您说声抱歉!

**Reply by feuyeux** [reply]
问题我多次回复过了 标准版启动闪退 beta版无法授权
要么你们把标准版问题修复掉 要么你们把flutter版本强化下成为标准版
环境 Win11最新版

**Comment by MaxMKMax** [comment]
@feuyeux 您好，谢谢您的反馈。蓝灯已经在正视这个问题了。 我们正在修复问题和进行优化。请您耐心等待。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Comment by feuyeux** [comment]
我今天升级了标准版 版本号 7.9.4 (20240820.152919) 解决了闪退问题 希望有和我相同问题的同胞 能同样解决。
![7.9.4 (20240820.152919)](https://github.com/user-attachments/assets/8dce8318-5a7f-4dfb-8913-d17100fc2cad)

![win11](https://github.com/user-attachments/assets/6fad0b1e-2bf5-46d7-ac73-c0e34be29985)
